# Hekateros hekateros_control/WriteGpio service

Gpio[] gpio     # GPIO pin state(s) to write
---
uint8[] state   # pin states(s) after write
